Dockerfile

```diff
@@ -10,4 +10,5 @@ ADD scm-source.json /scm-source.json
 EXPOSE 8080
 
 # run the server when a container based on this image is being run
-ENTRYPOINT java $JAVA_OPTS -Dloader.path=/plugins -Djava.security.egd=file:/dev/./urandom -jar nakadi.jar
\ No newline at end of file
+ENTRYPOINT java $JAVA_OPTS $(appdynamics-agent) -Dloader.path=/plugins -Djava.security.egd=file:/dev/./urandom -jar \
```

Comment:
@v1ctor we use "zalando/openjdk" Docker base image that's why the image already has necessary env for that (at least they say that)